src/python/build_management/build_manager.py

```diff
@@ -864,6 +864,49 @@ def setup(self):
     return True
 
 
+class CuttlefishKernelBuild(RegularBuild):
+  """Represents a Android Cuttlefish kernel build."""
+
+  _IMAGE_FILES = ('bzImage', 'initramfs.img')
+
+  def setup(self):
+    """Android kernel build setup."""
+    from platforms.android import adb
+
+    result = super().setup()
+    if not result:
+      return result
+
+    # Download syzkaller binary folder.
+    if not environment.get_value('ANDROID_EMULATOR_BUCKET_PATH'):
```

Comment:
let's not overload ANDROID_EMULATOR_BUCKET_PATH. Call this SYZKALLER_BUCKET_PATH instead.